packages/web/src/common/store/user-list/supporting/sagas.ts

```diff
@@ -23,7 +21,6 @@ import { watchSupportingError } from 'common/store/user-list/supporting/errorSag
 import { createUserListProvider } from 'common/store/user-list/utils'
 const { getId, getUserList, getUserIds } = supportingUserListSelectors
 const { getSupportingError } = supportingUserListActions
-const { setSupportingForUser } = tippingActions
 const { getUser } = cacheUsersSelectors
 
 type SupportingProcessExtraType = {
```

Comment:
do we still need this type? or should it be renamed?

Reply:
good find, likely not